docker-compose.yml

```diff
@@ -45,6 +45,7 @@ services:
       - rra
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+      - /data:/data
```

Comment:
I don't think this is needed.
This docker file is for the offline version which reads from the `offline` config file. I think it's ok for it to be contained inside the docker.